app/models/student_loans/eligibility.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module StudentLoans
+  class Eligibility < ApplicationRecord
+    self.table_name = "student_loans_eligibilities"
+
+    enum qts_award_year: {
+      "before_2013": 0,
+      "2013_2014": 1,
+      "2014_2015": 2,
+      "2015_2016": 3,
+      "2016_2017": 4,
+      "2017_2018": 5,
+      "2018_2019": 6,
+      "2019_2020": 7,
+    }, _prefix: :awarded_qualified_status
+
+    validates :qts_award_year, on: [:"qts-year"], presence: {message: "Select the academic year you were awarded qualified teacher status"}
```

Comment:
Do we no longer need to validate on `:submit`?

Reply:
Ah, good spot, we should definitely be validating the presence on `:submit`!